release_notes.md

```diff
@@ -4,5 +4,6 @@
 
 * Fix message loss bug in ContinueAsNew scenarios ([Azure/durabletask#544](https://github.com/Azure/durabletask/pull/544))
 * Allow custom connection string names when creating a DurableClient in an ASP.NET Core app (external app) (#1895)
+* Improve handling of deleted/disabled/renamed orchestrator and activity functions (#1901)
```

Comment:
I think we should be a bit more explicit here.

Proposal:

* Instead of endlessly retrying deleted, disabled, or renamed orchestrator and activity functions, we will now fail any in-flight executions if an active function with that name no longer exists.